src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1931,19 +1940,18 @@ public boolean isCheckboxChecked(String checkboxClass, String checkboxValue, int
         return checkbox.isSelected();
     }
 
-    // Only to be used for Rank Recipient Questions
-    public void clickEnableMinRankRecipients(int questionNumber) {
-        By checkboxSelector = By.id("minRecipientsToBeRankedEnabled-" + questionNumber);
-        WebElement minNumberOfOptionsToRankCheckbox = browser.driver.findElement(checkboxSelector);
-        ThreadHelper.waitFor(1000);
+    // Only to be used for Rank Options Questions
```

Comment:
This comment is not very helpful standalone although it is true that `InstructorFeedbackEditPage` is getting too large. If this is to be done, the other questions should have similar comments.